src/sugar/autofill.ts

```diff
@@ -46,6 +50,9 @@ async function autofill<T extends Transaction>(
   if (tx.LastLedgerSequence == null) {
     promises.push(setLatestValidatedLedgerSequence(client, tx))
   }
+  if (tx.TransactionType === 'AccountDelete') {
+    promises.push(checkAccountDeleteBlockers(client, tx as AccountDelete))
```

Comment:
Do we need this `as AccountDelete` there?

Reply:
I typed the method as requiring an `AccountDelete`, but I could type it as just `Transaction` and then the casting wouldn't be needed